File: loop_transformations/sources/apply_syntax_equivalent.1.c
/*
* @@name:	apply_syntax_equivalent.1
* @@type:	C
* @@operation:	compile
* @@expect:	success
* @@version:	omp_5.1
*/
void equivalent(double A[100])
{
   #pragma omp unroll
   for (int i1 = 0; i1 < 25; ++i1)
   for (int i2 = 0; i2 <  4; ++i2) {
      int i = i1 * 4 + i2;
      A[i] = A[i] + 1;
   }
}

File: loop_transformations/sources/apply_syntax_equivalent.1.f90
! @@name:       apply_syntax_equivalent.1
! @@type:       F-free
! @@operation:  compile
! @@expect:     success
! @@version:    omp_5.1
subroutine equivalent(A)
   implicit none
   double precision :: A(0:99)
   integer          :: i1,i2, i

   !$omp unroll
   do i1=0,24
   do i2=0, 3
      i = i1 * 4 + i2
      A(i) = A(i) + 1 
   enddo; enddo

end subroutine
